src/resources/postcss/datepicker.pcss

```diff
@@ -1,3 +1,23 @@
+.ui-datepicker {
+	font-family: "Verdana", "Helvetica Neue", "Helvetica", "Arial", sans-serif;
+
```

Comment:
Could we remove the empty line and alphabetize the properties for `.ui-datepicker`?